File: fluoxetine_backstage/src/main/resources/static/assets/js/service/companyService.js
app.service('companyService',function ($http) {

    this.search = function (entity,page,size) {
      return $http.post('/company/search?page='+page+'&size='+size,entity);
    };

    this.add = function (entity) {
      return $http.post('/company/add',entity);
    };

    this.findOne = function (id) {
        return $http.get('/company/findOne?id='+id);
    };

    this.update = function (entity) {
        return $http.post('/company/update',entity);
    };

    this.delete = function (ids) {
        return $http.get('/company/delete?ids='+ids);
    }
});

File: fluoxetine_backstage/src/main/java/cn/fluoxetine/fluoxetinebackstage/service/CompanyService.java
package cn.fluoxetine.fluoxetinebackstage.service;

import cn.fluoxetine.fluoxetinebackstage.pojo.FtbCompany;
import cn.fluoxetine.fluoxetinebackstage.pojo.PageResult;

/**
 * @author 11699
 * @date 2020/2/11 - 15:00
 */
public interface CompanyService {

    /**
     * 搜索
     * @param company
     * @param pageNum
     * @param pageSize
     * @return
     */
    public PageResult search(FtbCompany company,int pageNum,int pageSize);

    /**
     * 查找
     * @param id
     * @return
     */
    public FtbCompany findOne(Integer id);

    /**
     * 新增
     * @param company
     */
    public void add(FtbCompany company);

    /**
     * 修改
     * @param company
     */
    public void update(FtbCompany company);

    /**
     * 删除
     * @param ids
     */
    public void delete(Integer[] ids);
}
